packages/shared/components/inputs/Amount.svelte

```diff
@@ -100,7 +116,7 @@
 <amount-input class:disabled class="relative block {classes}" on:keydown={handleKey}>
     <Input
         {error}
-        label={label || locale('general.amount')}
+        label={fiatAmount ? `${fiatAmount} ${profileCurrency}` : label || locale('general.amount')}
```

Comment:
This would probably be better as `{fiatAmount !== null` otherwise it doesn't show when the amount is zero. e.g. `111i` still says `Amount`